incubator/hnc/pkg/validators/object_test.go

```diff
@@ -0,0 +1,64 @@
+package validators
+
+import (
+	"context"
+	"testing"
+
+	. "github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+
+	api "github.com/kubernetes-sigs/multi-tenancy/incubator/hnc/api/v1alpha1"
+	"github.com/kubernetes-sigs/multi-tenancy/incubator/hnc/pkg/forest"
+)
+
+func TestObject(t *testing.T) {
```

Comment:
I'd call this TestInheritedFromLabel since you're not testing _arbitrary_ properties of the object. Then each subtest name can be shorter (ie "label doesn't exist").